primitives/staking/src/offence.rs

```diff
@@ -142,19 +142,28 @@ pub trait OnOffenceHandler<Reporter, Offender> {
 	/// Zero is a valid value for a fraction.
 	///
 	/// The `session` parameter is the session index of the offence.
+	///
+	/// The receiver might decide to not accept this offence. In this case, the call site is
+	/// responsible for queuing the report and re-submitting again.
```

Comment:
If that's common we should probably extract this logic into some kind of a wrapper.

Reply:
We have deferring in quite a few places (in Staking and slashing) but I can't see how you can abstract them away, since IMO each has its own taste and details. Do you have a more concrete example of what you are suggesting here?